src/vendors/actions.ts

```diff
@@ -1,10 +1,12 @@
-import { Vendor, Vendors } from 'src/vendors/types'
+import { Vendor, Vendors, VendorWithLocation } from 'src/vendors/types'
 
 export enum Actions {
   FETCH_VENDORS = 'VENDORS/FETCH_VENDORS',
   SET_VENDORS = 'VENDORS/SET_VENDORS',
   SET_LOADING = 'VENDORS/SET_LOADING',
   SET_CURRENT_VENDOR = 'VENDORS/SET_CURRENT_VENDOR',
+  SET_FILTERED_VENDORS = 'VENDORS/SET_FILTERED_VENDORS',
+  SET_SEARCH_QUERY = 'VENDORS/SET_SEARCH_QUERY',
```

Comment:
This search bar is used for the entire map, not just vendors. I would create a new reducer for `map` general actions like this.